### PR TITLE
SC-3686 - Mail-Notification-Only-Frontend

### PR DESCRIPTION
**Pull request by ABialoskorski**
# Issue: [SC-3686](https://ticketsystem.schul-cloud.org/browse/SC-3686)

## Description
PR to frontend part of SC-3686

## Screenshot

image

### CODE REVIEW

File: src/pages/administration/students/index.vue [hunk 1]
```diff
@@ -50,10 +50,23 @@
...
 					<base-icon
 						source="material"
-						icon="close"
+						icon="cancel"
 						color="var(--color-danger)"
 					/>
 				</span>
+				<span
+					v-else-if="data && Object.keys(data).length === 0"
+					class="text-content"
```
Comment (ABialoskorski) [inline src/pages/administration/students/index.vue:59]: due to SC-4103

File: src/pages/administration/teachers/index.vue [hunk 2]
```diff
@@ -45,10 +45,23 @@
...
 					<base-icon
 						source="material"
-						icon="close"
+						icon="cancel"
 						color="var(--color-danger)"
 					/>
 				</span>
+				<span
+					v-else-if="data && Object.keys(data).length === 0"
+					class="text-content"
```
Comment (ABialoskorski) [inline src/pages/administration/teachers/index.vue:54]: due to SC-4103

**Comment by dominikbruechner** [comment]
could you also add screenshots of the changed screens please, thanks :)

**Comment by ABialoskorski** [comment]
> could you also add screenshots of the changed screens please, thanks :)

sure, done :)

**Comment by ABialoskorski** [comment]
> as there is an option in the UI to create a user without sending an email, shouldn't there be any check that the mail was actually send?
> As far as I understand the current implementation just shows "mail send" for every item that does not have a consent status which may be missleading.

good idea, should someone add that to ticket?

**Comment by adrianjost** [comment]
> should someone add that to ticket?
@ABialoskorski yes, can you do this? It may also require some backend work, because I think the mail send is currently not tracked by the backend

**Comment by haniapalac** [comment]
Here is the new screenshot of the changes in the UI:
image

**Comment by haniapalac** [comment]
@schul-cloud-bot update screenshots

**Comment by haniapalac** [comment]
@schul-cloud-bot update screenshots

File: locale/de.json [hunk 3]
```diff
@@ -230,9 +233,9 @@
 	"pages.administration.students.index.tableActions.email": "Registrierungslinks per E-Mail versenden",
 	"pages.administration.students.index.tableActions.qr": "Registrierungslinks als QR-Code ausdrucken",
 	"pages.administration.students.index.title": "Schüler:innen verwalten",
-	"pages.administration.students.legend.icon.danger": "Keine Einverständniserklärungen vorhanden",
```
Comment (tofixx) [inline locale/de.json:233]: are all removed values no more in use?

File: locale/en.json [hunk 4]
```diff
@@ -5,6 +5,8 @@
 	"common.actions.save": "",
 	"common.actions.scrollToTop": "Scroll up",
 	"common.labels.consent": "Consent",
+	"common.labels.consent.parents": "Parent",
```
Comment (tofixx) [inline locale/en.json:8]: isn't this plural hgere and below?

File: src/assets/icons/email.svg [hunk 5]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24"><path d="M0 0h24v24H0V0z" fill="none"/><path d="M22 6c0-1.1-.9-2-2-2H4c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6zm-2 0l-8 5-8-5h16zm0 12H4V8l8 5 8-5v10z"/></svg>
```
Comment (tofixx) [inline src/assets/icons/email.svg:1]: empty line missing at EOF

**Comment by haniapalac** [comment]
The same work is done in another ticket (https://ticketsystem.schul-cloud.org/browse/SC-6173), so this PR can be closed

**Pull request closed without merging**